docs/spec_draft.md

```diff
@@ -705,6 +706,64 @@ lexicographic comparison on the (real, imaginary) pairs.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.multiply
+
+### Semantics
+
+Performs element-wise product of two tensors `lhs` and `rhs` and produces a
+`result` tensor. For integer element types, if the element-wise product has an
+unsigned/signed overflow/underflow, the result is implementation-defined and one
+of the following:
+
+  * mathematical result modulo $2^n$, where n is the bit width of the result,
+  for unsigned overflow/underflow. For signed integer overflow/underflow, wraps
+  the result around the representable range $[-2^{n-1},\ \ 2^{n-1} - 1]$.
+  * saturation to $2^{n-1} - 1$ (or $-2^{n-1}$) for signed overflow (or signed
+  underflow) and saturation to $2^n - 1$ (or $0$) for unsigned overflow (or
+  unsigned underflow).
+
+For floating-point element types, it implements the `multiplication` operation from
+the IEEE-754 specification.
+
+For complex element types, the operation follows multiplication semantics for
```

Comment:
How about

The multiplication of two complex constants (a,b) and (c.d) is given by `(a*c - b*d, a*d + b*c)`, where `*`, `+`, and `-` respectively implements the `multiplication`, `addition`, and `subtraction` from the IEEE-754 specification.


This is in sync with how we represent complex constants.

Reply:
For other ops on complex types, e.g. `exponential`, `sin`, etc, we just say "For complex element types, it computes a complex exponential, with corner cases TBD". Any reason not to do that too?

Reply:
On second thought, I think the multiplication of two complex number is a well know concept and we should not mention it for the same reason we skip mentioning addition of complex numbers.

Reply:
I agree with that. Something like
```
For complex element types, it computes a complex multiplication, with corner cases TBD.
```

Reply:
Done

Reply:
Please review and resolve.